src/index.ts

```diff
@@ -16,11 +16,21 @@
 
 import * as firebase from './default-namespace';
 
-// Register the Database service
-// For historical reasons, the database code is included as minified code and registers itself
-// as a side effect of requiring the file.
-/* tslint:disable:no-var-requires */
-// require('./database/database');
-/* tslint:enable:no-var-requires */
+// Only Node.js has a process variable that is of [[Class]] process
+if (Object.prototype.toString.call(global.process) !== '[object process]') {
```

Comment:
Not sure If the package will ever run in an env without `global` global. 
If yes, we should do `if (global && Object.prototype.toString.call(global.process) !== '[object process]')`

Reply:
Looks like browsers don't have it. Therefore I'm checking for `process` instead of `global` now. Check adapted from https://github.com/iliakan/detect-node/blob/master/index.js